tutorials/time-freq/plot_ssvep.py

```diff
@@ -0,0 +1,786 @@
+"""
+=======================================
+Basic analysis of an SSVEP/vSSR dataset
+=======================================
+
+In this tutorial we compute the frequency spectrum and quantify signal-to-noise
+ratio (SNR) at a target frequency in EEG data recorded during fast periodic
+visual stimulation (FPVS).
+Extracting SNR at stimulation frequency is a simple way to quantify frequency
+tagged responses in MEEG (a.k.a. steady state visually evoked potentials,
+SSVEP, or visual steady-state responses, vSSR in the visual domain,
+or auditory steady-state responses, ASSR in the auditory domain).
+
+DATA:
+
+We use a simple example dataset with frequency tagged visual stimulation:
+N=2 participants observed checkerboards patterns inverting with a constant
+frequency of either 12Hz of 15Hz. 10 trials of 20s length each.
+32ch wet EEG was recorded.
+
+Data can be downloaded at https://osf.io/7ne6y/
+Data format: BrainVision .eeg/.vhdr/.vmrk files organized according to BIDS
+standard.
```

Comment:
The data download location is probably not needed. The data format specs like being in BIDS could probably live in `datasets_index.rst` (where an entry is needed anyway, e.g., with a link ref like `ssvep-dataset`) so that this tutorial can focus on the processing mechanics and justification:

https://github.com/mne-tools/mne-python/blob/main/doc/overview/datasets_index.rst

Reply:
+1 to link to datasets_index page.

btw it also does not render well.